### ISSUE TITLE
Any Finetuned BC Model with English Dataset ?

### ISSUE THREAD

**Issue by mssdot** [inokoj/VAP-Realtime#7]
any finetuned model for backchannel trained with English dataset?

**Comment by inokoj** [comment]
Yes, we are trying to make it.
Do you suggest any English datasets containing the annotation of backchannels?

**Comment by mssdot** [comment]
I am not sure about the annotations, maybe you could try using the CALLHOME or CALLFRIEND datasets.

Even if I want to fine-tune this model on the BC dataset, is the same training setup sufficient?

I didn’t find any **loss** related to BC during backpropagation in the script, except for the additional linear layers added to the model architecture.

Also, how should the data be prepared for training the BC model? since training readme , annotations are not mentioned in data preparation.

It would be really helpful if you could provide separate training scripts or readme for training the BC model. Thanks

**Comment by inokoj** [comment]
Thank you. You're right—the current repository does not yet include the training program for backchannel prediction. I will add that shortly. The data should include backchannel segmentation alongside the original utterance segment annotations.

**Comment by mssdot** [comment]
thanks for response
